### PR TITLE
Implement API to Start BFT via RPC call

### PR DESCRIPTION
**Pull request by talekhinezh**
* Added `consensus.start_on_boot` config boolean property to determine whether to start consensus on boot
* Added `BFT.start` JSON-RPC call to manually start consensus

### CODE REVIEW

File: radixdlt/src/main/java/com/radixdlt/consensus/ChainedBFT.java [hunk 1]
```diff
@@ -25,13 +25,13 @@
...
 import io.reactivex.rxjava3.core.Scheduler;
+import io.reactivex.rxjava3.observables.ConnectableObservable;
 import io.reactivex.rxjava3.schedulers.Schedulers;
-import java.util.Objects;
 import java.util.concurrent.Executors;
 
 /**
- * A three-chain BFT. The inputs are events via rx streams from a pacemaker and
- * a network.
+ * Subscription Manager (Start/Stop0 to the processing of BFT events under
```
Comment (flotothemoon) [inline radixdlt/src/main/java/com/radixdlt/consensus/ChainedBFT.java]: Parse error at line 33
(unmatched opening bracket)

**Review by flotothemoon — APPROVED** [review]

File: radixdlt/src/integration/java/com/radixdlt/consensus/simulation/BFTNetworkSimulation.java [hunk 2]
```diff
@@ -181,12 +182,12 @@ public PacemakerImpl getPacemaker(ECKeyPair keyPair) {
 		return pacemakers.get(keyPair);
 	}
 
-	public TestEventCoordinatorNetwork getUnderlyingNetwork() {
-		return underlyingNetwork;
+	public void start() {
+		this.bfts.forEach((e, bft) -> bft.start());
```
Comment (clstrfsck) [inline radixdlt/src/integration/java/com/radixdlt/consensus/simulation/BFTNetworkSimulation.java]: Maybe `this.bfts.values().foreach(ChainedBFT::start)`.

File: radixdlt/src/main/java/org/radix/Radix.java [hunk 3]
```diff
@@ -143,25 +145,20 @@ public static void start(RuntimeProperties properties) {
...
-					log.error("BFT Unexpected Error", e);
-					System.exit(-1);
-				}
-			);
-
 		SubmissionControl submissionControl = globalInjector.getInjector().getInstance(SubmissionControl.class);
 		AtomToBinaryConverter atomToBinaryConverter = globalInjector.getInjector().getInstance(AtomToBinaryConverter.class);
 		LedgerEntryStore store = globalInjector.getInjector().getInstance(LedgerEntryStore.class);
-		RadixHttpServer httpServer = new RadixHttpServer(store, submissionControl, atomToBinaryConverter, universe, serialization, properties, localSystem, addressBook);
+		RadixHttpServer httpServer = new RadixHttpServer(bft, store, submissionControl, atomToBinaryConverter, universe, serialization, properties, localSystem, addressBook);
```
Comment (clstrfsck) [inline radixdlt/src/main/java/org/radix/Radix.java:151]: Eventually we will just pass all objects into RadixHttpServer.

File: radixdlt/src/main/java/org/radix/api/jsonrpc/RadixJsonRpcServer.java [hunk 4]
```diff
@@ -157,6 +168,11 @@ private JSONObject handle(JSONObject jsonRpcRequest) {
 			final String method = jsonRpcRequest.getString("method");
 			final Object paramsObject = jsonRpcRequest.get("params");
 			switch (method) {
+				case "BFT.start":
+					chainedBFT.start();
+					result = new JSONObject()
+                        .put("response", "success");
```
Comment (clstrfsck) [inline radixdlt/src/main/java/org/radix/api/jsonrpc/RadixJsonRpcServer.java]: Spaces.  Could also easily argue that everything else is wrong, and this is the only correct line.

File: radixdlt/src/main/java/org/radix/Radix.java [hunk 5]
```diff
@@ -32,6 +33,7 @@
 import com.radixdlt.universe.Universe;
 import com.radixdlt.utils.Bytes;
 
+import io.reactivex.rxjava3.observables.ConnectableObservable;
```
Comment (clstrfsck) [inline radixdlt/src/main/java/org/radix/Radix.java]: I don't think this import is used, but checkstyle didn't seem to complain about it?

**Review by clstrfsck — APPROVED** [review]

File: radixdlt/src/main/java/org/radix/Radix.java (hunk 5 above)
Comment (talekhinezh) [reply radixdlt/src/main/java/org/radix/Radix.java]: yeah it doesn't seem like Radix.java is being checked

File: radixdlt/src/main/java/org/radix/Radix.java (hunk 3 above)
Comment (talekhinezh) [reply radixdlt/src/main/java/org/radix/Radix.java:151]: yup, need to turn it injectable but a bit of refactoring still needed

**Pull request merged**